src/Microsoft.ML.Core/Data/Range.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML.CommandLine;
+
+namespace Microsoft.ML.Data
+{
+    public class Range
```

Comment:
We can't be adding a public type named `Range`. It conflicts with the new `System.Range`.

[https://github.com/dotnet/designs/pull/48](https://github.com/dotnet/designs/pull/48)

This would be equivalent to naming one of our new classes `String` or `Object`.